docs/CHANGELOG.asciidoc

```diff
@@ -39,6 +40,309 @@ This section summarizes the changes in each release.
+* We updated the snapshot policy name field in Delete phase of index lifecycle policy. This component now display a list of existing snapshot policies and warns the user if their input doesn't match any existing policies {kibana-pull}70627[#70627]
+Maps::
+* Displays ranged-data with bands {kibana-pull}60570[#60570]
+* Adds styling and tooltip support to mapbox mvt vector tile sources {kibana-pull}64488[#64488]
+* Allows adding multiple layers {kibana-pull}67544[#67544]
+* Enables gridding/clustering/heatmaps for geo_shape fields {kibana-pull}67886[#67886]
+* Security layer wizards {kibana-pull}68290[#68290]
+* Surface geo_shape clustering gold feature {kibana-pull}68666[#68666]
+* Layer wizard select re-design {kibana-pull}69313[#69313]
+* Shows vector tile labels on top {kibana-pull}69444[#69444]
+* Choropleth layer wizard {kibana-pull}69699[#69699]
+* Increases DEFAULT_MAX_BUCKETS_LIMIT to 65535 {kibana-pull}70313[#70313]
+* Shows joins disabled message {kibana-pull}70826[#70826]
+* Exposes registerLayerWizard and registerSource in maps plugin start {kibana-pull}71553[#71553]
+Metrics::
+* Adds support for multiple groupings to Metrics Explorer (and Alerts) {kibana-pull}66503[#66503]
+* Adds sorting for name and value to Inventory View {kibana-pull}66644[#66644]
+* Changes Metric Threshold Alert charts to use bar charts {kibana-pull}66672[#66672]
+* Allows users to configure Inventory View palettes {kibana-pull}66948[#66948]
+* Adds timestamp context variable {kibana-pull}67482[#67482]
+* Adds back context variables with descriptions {kibana-pull}67487[#67487]
+* Adds overrides to Snapshot API to support alert previews {kibana-pull}68125[#68125]
+* Enhances Inventory View Tooltips {kibana-pull}69757[#69757]
+* UX improvements for saved views {kibana-pull}69910[#69910]
+* Register function for Observability homepage {kibana-pull}70529[#70529]
+* Adds framework for recovery messaging to metric threshold alerts (non-functional) {kibana-pull}65339[#65339]
+* Adds preview feature for metric threshold alerts {kibana-pull}67684[#67684]
+* Adds inventory alert preview {kibana-pull}68909[#68909]
+* Prefills alerts from the global dropdown {kibana-pull}68967[#68967]
+* Adds context.reason and alertOnNoData to Inventory alerts {kibana-pull}70260[#70260]
+Monitoring::
+* Collects number of visualization saved in the past 7, 30 and 90 days {kibana-pull}67865[#67865]
+* Out of the box alerting {kibana-pull}68805[#68805]
+Operations::
+* Node options from cfg file for production {kibana-pull}62468[#62468]
+* Creates Linux aarch64 package {kibana-pull}69165[#69165]
+Platform::
+* Switches to core application service {kibana-pull}63443[#63443]
+* Lens editor auto refresh {kibana-pull}65868[#65868]
+* Adds simple variables to workpads {kibana-pull}66139[#66139]
+* Adds docLinks to CoreSetup {kibana-pull}66631[#66631]
+* New Enterprise Search Kibana plugin {kibana-pull}66922[#66922]
+* Workplace Search in Kibana MVP {kibana-pull}70979[#70979]
```

Comment:
Not sure if we have one, but these should probably be in a Enterprise Search section?

Reply:
@rayafratkina are we in a position to add an Enterprise Search section to the Kibana Release Notes?